packages/react/tsconfig.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "module": "ESNext",
     "outDir": "dist"
-  }
+  },
+  "include": ["./src"]
 }
```

Comment:
Do we still need the ditinction between `tsconfig.json` and `tsconfig.base.json`?

Reply:
Another strange quirk I deferred to after we agree on a path for this - seems the base TS config `includes` is being picked up by preconstruct